phpstan.neon

```diff
@@ -12,3 +12,7 @@ parameters:
         Doctrine\ORM\Query\Parser:
             - syntaxError
     phpVersion: 70100
+
+    ignoreErrors:
+        # The class was added in PHP 8.1
+        - '/^Attribute class ReturnTypeWillChange does not exist.$/'
```

Comment:
Should the CI job be updated to run on PHP 8? That way you would have that error message.

Reply:
Sure, why not.